docs/src/sections/commercialization.adoc

```diff
@@ -1,3 +1,99 @@
 === Komercjalizacja projektu
 
-TODO
+Projekt w swoim założeniu jest dosyć specyficzny. Posiada zaawansowane funkcjonalności takie jak płatność w sieci
+Lightning, generowanie tokenów czy też podpisywanie transakcji w multisig w aplikacji mobilnej. Nie będzie on
+natomiast posiadać wielu podstawowych, o wiele prostszych rozwiązań. Przykładowo nie będzie możliwości standardowych
+płatności w Bitcoinach, a jedynie poprzez sieć Lightning. Na rynku jest również dostępnych wiele innych kryptowalut
+innych niż sam Bitcoin, co również stawia nas na przegranej pozycji. Produkt skierowany jest do bardzo wąskiej grupy
+odbiorców (przede wszystkim projekt _Ingenious_), dlatego zdecydowano się na wydanie projektu w modelu open-source na
+licencji MIT. Zainteresowana osoba będzie mogła sprawdzić projekt, a także wpływać na jego rozwój za darmo. Jedynym
+sposobem do komercjalizacji jest oferowanie klientom biznesowym płatnego wsparcia technicznego. Obejmować ono może
+pomoc przy wdrożeniu oraz utrzymanie w środowisku produkcyjnym. Licencja MIT pozwala w razie potrzeby na rozwój
+projektu przy zmianie licencji, więc drzwi do szerszej komercjalizacji zostaną otwarte.
+
+[NOTE]
+Dane bazują przede wszystkim na dwóch raportach firmy _crypto.com_: _Overview of crypto payment_
+<<crypro_payment_raport>> oraz _Crypto market sizing_ <<crypto_market_sizing>>.
+Wykresy znajdujące się w następnych podrozdziałach pochodzą również z tychże raportów.
+
+==== Analiza rynku
+
+Proponowane rozwiązanie najbardziej pasuje do rynku płatności w kryptowalutach (_Crypto payments_ bądź _Crypto payment
+gateways/processors_). Jest to stosunkowo nowy rynek, który dynamicznie się rozwija. Na rynku jest wiele komercyjnych
+rozwiązań takich jak opisane wcześnie _Coinbase Commerce_ i _Bitrefill_, ale również popularne są _BitPay_ czy
+_Coingate_. Z rozwiązań open-source najpopularniejszy jest natomiast BTCPayServer. Ze zgromadzonych przez zespół
+projektowy danych wynika, że rozwiązanie _Coinbase_ ma największy udział na rynku na poziomie 90%. Wynika to z
+ogólnodostępnych danych dostępnych na stronach internetowych poszczególnych rozwiązań. Tych informacji nie należy
+traktować bardzo poważnie, gdyż każda firma może dowolnie manipulować prezentowanymi informacjami.
+
+Według raportu wykonanego przez serwis _crypto.com_ suma płatności w roku 2020 przy pomocy kryptowalutach wyniosła od
+15,66 do 16,52 miliardów dolarów. Daje to aktualnie 0.4% wszystkich przychodów z branży _ecommerce_. Może się to
+wydawać małą kwotą, natomiast trzeba pamiętać o nowości i innowacyjności kryptowalut. Jest to stosunkowo młoda
```

Comment:
innowacyjność kryptowalut ok, ale nowość kryptowalut moim zdaniem dziwnie brzmi. 
jest nowoscia na rynku/ dosyć krótko na rynku, przydałoby się trochę zredagować to zdanie

Reply:
done